ParisBNF/abb/BNFabb57.xml

```diff
@@ -189,124 +190,136 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <additions>
               <list>
                 <item xml:id="a1">
-                  <locus target="#1r"/>
-                  <desc type="OwnershipNote" xml:lang="gez"/>
+                  <locus target="#31rb"/>
+                  <desc type="OwnershipNote" xml:lang="gez">An extended ownership note, mentioning the
+                    monastery Dāgā ʾƎsṭifānos</desc>
                   <q xml:lang="gez"/>
-                  <note>The note is written in a crude hand mentions. It seems to mention the
-                    monastery of <placeName ref="ETH1348Daga">Dāgā ʾƎsṭifānos</placeName> of the
-                    owner of the manuscript. </note>
+                  <note>The note is written in a crude hand.</note>
```

Comment:
could you delete `<note>` and insert the content into `<desc>`? It might not be visualised, and it's better to avoid many notes
also below